docs/articles/getting-started.md

```diff
@@ -1,37 +1,37 @@
----
-uid: getting-started
----
-
 # Getting Started
 
-## 1. How to install Colore
+***Important***: *This page is for v6.0.0 of Colore, which is currently only available as a pre-release. For the v5.2 getting started guide (not recommended), [[click here|Getting-started-(v5.2)]].*
+
 [![NuGet version][ngverbadge]][ng]
 [![MyGet version][mgverbadge]][mg]
 
-The easiest way to include Colore into your project is to right click on your C# Project in the Visual Studio Solution Explorer and Choose "Manage NuGet Packages..."
+## 1. Installing Chroma SDK
+Razer Chroma SDK is a C++-tool which allows us to access Chroma devices. It is the backend (or native/unmanaged code) behind Colore. It is expected to be automatically installed with Synapse once you plug in a Chroma device, but in some cases that might not work (see [#263](https://github.com/chroma-sdk/Colore/issues/263#)). In this case you can download the so called [Chroma SDK Core](https://assets.razerzone.com/dev_portal/downloads/Razer_Chroma_SDK_Core_v1.10.6.exe) which is just the SDK without Synapse.
```

Comment:
I think we should hold off on linking to an installer here, until we have word from Razer on the installation process. The last I heard from them is that the Chroma SDK is to only be installed via Synapse, no other method is currently officially supported.

Reply:
They offically provide this tool on the download page. Maybe I can mention that you can download it as a last resort.

Reply:
Where is the page with that link? I'm unable to find it looking at the dev pages. Would be better to link to the page that has the download link rather than a direct link to a specific version, to prevent it getting out of date.